VAMobile/src/screens/OnboardingCarousel/GenericOnboarding/GenericOnboarding.test.tsx

```diff
@@ -1,34 +1,28 @@
 import 'react-native'
 import React from 'react'
-import 'jest-styled-components'
-import { ReactTestInstance } from 'react-test-renderer'
 
-import { context, render, RenderAPI } from 'testUtils'
+import { context, render } from 'testUtils'
 import GenericOnboarding from './GenericOnboarding'
-import { VAIcon } from 'components'
+import { screen } from '@testing-library/react-native'
```

Comment:
This can be grouped with the imports above

Reply:
updated